dev-tools/scripts/releaseWizard.yaml

```diff
@@ -280,10 +280,10 @@ groups:
       voting rules, create a PGP/GPG key for use with signing and more. Please familiarise
       yourself with the resources listed below.
     links:
-    - http://www.apache.org/dev/release-publishing.html
-    - http://www.apache.org/legal/release-policy.html
-    - http://www.apache.org/dev/release-signing.html
-    - https://wiki.apache.org/lucene-java/ReleaseTodo
+    - https://infra.apache.org/release-publishing.html
+    - https://www.apache.org/legal/release-policy.html
+    - https://infra.apache.org/release-signing.html
+    - https://cwiki.apache.org/confluence/display/LUCENE/ReleaseTodo
```

Comment:
Should we just copy this over and have a 'Solr' link here instead?

Reply:
Maybe? The page itself is marked deprecated to begin with.